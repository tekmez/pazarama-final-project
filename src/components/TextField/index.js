import React from "react";

function TextField(props) {
  const { type, name, value, placeholder, onChange, className, onKeyPress } =
    props;
  return (
    <input
      type={type}
      name={name}
      value={value}
      placeholder={placeholder}
      onChange={onChange}
      className={className}
      onKeyPress={onKeyPress}
    />
  );
}

export default TextField;
