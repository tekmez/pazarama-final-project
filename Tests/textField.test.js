import React from "react";
import renderer from "react-test-renderer";
import TextField from "../src/components/TextField/index";
test("Text Field Component", () => {
  const tree = renderer
    .create(
      <TextField user={{type: "text", name: "Test", value: "value", placeholder: "holder", className: "classname", onChange: (()=>{}), onKeyPress: (()=>{}) }} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});