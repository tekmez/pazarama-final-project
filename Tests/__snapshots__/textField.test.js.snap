// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Field Component 1`] = `<input />`;
